libshaderc_spvc/include/spvc/spvc.hpp

```diff
@@ -271,27 +271,27 @@ class CompileOptions {
   CompileOptions& operator=(const CompileOptions& other) = delete;
   shaderc_spvc_compile_options_t options_;
 
-  friend class Compiler;
+  friend class Context;
```

Comment:
Why class CompileOptions is friends with class Context? (I know it has nothing with your changes, I should have asked this before)

Reply:
I didn't write that code, but it looks like it is due to wanting to be able to have the context reach into the private options_ member to pass into the C API, but not wanting expose that to the public API. Context & Options do not have an inheritance relationship, so protected cannot be used to avoid this.

Given that this is C++, composition of interfaces or multiple inheritence, could have been used instead, to avoid using friends, but I feel that is probably a more verbose way of effectively doing the same thing. If we needed to do more then access a single member or if there were more OO machinery interacting with this, then it might be warranted.